### DISCUSSION TITLE
Chart legend bug

### DISCUSSION THREAD

**Discussion by Ultil** [getredash/redash#6231] [Q&A] [answered]
Hello,

Does anyone know how to stop teh legend covering the second Y axis. Example attached.

<img width="458" alt="chart bug" src="https://github.com/getredash/redash/assets/140156248/1de3d315-f37f-4e4d-a548-93a145c57f1a">

Thank you in advance!

**Comment by gaecoli** [comment, marked as answer]
In the dashboard, you can try to adjust the size of the widget to solve it. If it is in the query, it is a style bug, and you need to adjust the display with a larger distance.

**Reply by Ultil** [reply]
Thank you very much

**Comment by Ultil** [comment]
answered
